lib/AccessControl/View/PoliciesTable/PoliciesTable.test.js

```diff
@@ -70,33 +90,48 @@ describe('PoliciesTable', () => {
         {
           row: 'first',
           rowIndex: 0,
-          expectedData: {
-            ...flattenedPolicies[0],
+          getExpected: () => ({
+            ...sortedByNameAsc[0],
             rowIndex: 0, // MCL adds this information in
-          }
+          }),
         },
         {
           row: 'third',
           rowIndex: 2,
-          expectedData: {
-            ...flattenedPolicies[2],
+          getExpected: () => ({
+            ...sortedByNameAsc[2],
             rowIndex: 2, // MCL adds this information in
-          }
-        }
-      ])('clicking the remove button for the $row row', ({ rowIndex, expectedData }) => {
+          }),
+        },
+      ])('clicking the remove button for the $row row', ({ rowIndex, getExpected }) => {
         beforeEach(async () => {
-          onRemove.mockClear();
+          onDoRemove.mockClear();
           await waitFor(async () => {
             await MultiColumnListCell({ row: rowIndex, columnIndex: 3 }).find(IconButton()).click();
           });
         });
 
         test('onRemove got called with the correct data', async () => {
           await waitFor(() => {
-            expect(onRemove).toHaveBeenCalledWith(expectedData);
+            expect(onDoRemove).toHaveBeenCalledWith(getExpected());
           });
         });
       });
+
+      // NEW: Clicking "Remove" header changes the sort column (branch meta.name !== sortOrder)
```

Comment:
We shouldn't litter the tests with `// NEW` comments, because they won't always be NEW